transcendence-app/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG NODE_VERSION=16.14.2
+ARG ALPINE_VERSION=3.15
+
+FROM node:${NODE_VERSION}-alpine${ALPINE_VERSION} AS development
+ENV NODE_ENV=development
```

Comment:
Just to get how this works. I understand this kind of two targets for whether you deploy the app as dev or prod?

Reply:
I've seen this is the multi stage thing you mentioned, I'll read about it, thanks!

Reply:
It's a pretty cool feature. You can have one Dockerfile and build various images(development, production, staging, CI, etc...). Also, it's possible to copy files from other stages or create images targeting a specific stage with the `--target` option.